CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * [FEATURE] - [Change the date time form fields to not use `form.datetime_select`](https://trello.com/c/91fUZw28/189-3-change-the-date-time-form-fields-to-not-use-formdatetimeselect)
 * [BUG] - [Style the links and highlights on the preview page](https://trello.com/c/DI8wiPw2/110-spike-style-the-links-and-highlights-on-the-preview-page)
 * [FEATURE] - [Session naming](https://trello.com/c/VQbfdtLM/222-8-session-naming)
+* [FEATURE] - [Add location input to where&when](https://trello.com/c/wdLqH6qT/228-1-add-location-input-to-wherewhen)
```

Comment:
You are adding this to a released release ;)

Reply:
I previously rebased a branch into itself. I am going for the full set.

For my next trick, referring to a pull request in its own comment

#154